src/sagemaker_pytorch_serving_container/torchserve.py

```diff
@@ -0,0 +1,212 @@
+# Copyright 2019-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+"""This module contains functionality to configure and start Torchserve."""
+from __future__ import absolute_import
+
+import os
+import signal
+import subprocess
+import sys
+
+import pkg_resources
+import psutil
+from retrying import retry
+
+import sagemaker_pytorch_serving_container
+from sagemaker_inference import default_handler_service, environment, logging, utils
+from sagemaker_inference.environment import code_dir
+
+logger = logging.get_logger(__name__)
```

Comment:
my bad - led you astray earlier here

```suggestion
logger = logging.getLogger(__name__)
```